### ISSUE TITLE
dbus-adc: GetText does not reply with the status in text

### ISSUE THREAD

**Issue by mpvader** [victronenergy/venus#152]
Just a minor annoyance, since gui etc all have there own translation of the enum into a text.

```
# dbus -y com.victronenergy.tank.builtin_adc4_di0 / GetText
value = {'Capacity': '0.200000003',
 'Connected': '1',
 'DeviceInstance': '20',
 'FluidType': '0',
 'Level': '',
 'Mgmt/Connection': 'Tank Level sensor input 1',
 'Mgmt/ProcessName': 'dbus-adc',
 'Mgmt/ProcessVersion': '1.13',
 'ProductId': '41313',
 'ProductName': 'Generic Tank Input',
 'Remaining': '',
 'Standard': '0',
 'Status': '1',             <--- This should have been 'Disconnected'
 'analogpinFunc': '1'}
```

**Comment by edevries-victron** [comment]
/FluidType and /Standard are also enums. Fix them as well?

**Comment by mpvader** [comment]
Yes

**Comment by edevries-victron** [comment]
Fixed with dbus-adc v1.14

**Comment by mpvader** [comment]
Venus v2.08~4
